SpectodaParser.js

```
@@ -1016,24 +1036,26 @@ export class TnglCompiler {
     switch (puctuation) {
-      case "{":
+      case '{':
         // push the current depth of the variable stack to the depth stack
         this.#const_scope_depth_stack.push(this.#const_declarations_stack.length);
         this.#let_scope_depth_stack.push(this.#let_declarations_stack.length);
         break;
 
-      case "}":
+      case '}':
         // pop the scope depth of the depth stack variable stack and set the variable stack to the previous depth
         const const_depth = this.#const_scope_depth_stack.pop();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)